tests/ragger/test_clone.py

```diff
@@ -0,0 +1,32 @@
+from web3 import Web3
+
+from ragger.backend import BackendInterface
+from ragger.firmware import Firmware
+from ragger.navigator import Navigator
+
+from test_sign import common
+
+
+# Values used across all tests
+ADDR = bytes.fromhex("5a321744667052affa8386ed49e00ef223cbffc3")
+BIP32_PATH = "m/44'/1001'/0'/0/0"
+NONCE = 68
+GAS_PRICE = 13
+GAS_LIMIT = 21000
+VALUE = 0.31415
+
+
+# Legacy Zemu etc with clone: Transfer on Ethereum clone app
```

Comment:
With the new test function name, this comment is kind of useless now.

Reply:
:stuck_out_tongue_winking_eye: 
Fixed